interface/python_binding/build.py

```diff
@@ -0,0 +1,197 @@
+# Copyright (c) 2023, Shukant Pal and Contributors
+# SPDX-License-Identifier: Apache-2.0
+
+from cffi import FFI
+import sys
+import os
+import unittest
+
+LIBKTX_INSTALL_DIR = os.getenv("LIBKTX_INSTALL_DIR")
+LIBKTX_INCLUDE_DIR = os.getenv("LIBKTX_INCLUDE_DIR")
+LIBKTX_LIB_DIR = os.getenv("LIBKTX_LIB_DIR")
+
+if os.name == 'nt':
+    if LIBKTX_INSTALL_DIR is None:
+        LIBKTX_INSTALL_DIR = 'C:\\Program Files\\KTX-Software'
+    if LIBKTX_INCLUDE_DIR is None:
+        LIBKTX_INCLUDE_DIR = LIBKTX_INSTALL_DIR + '\\include'
+    if LIBKTX_LIB_DIR is None:
+        LIBKTX_LIB_DIR = LIBKTX_INSTALL_DIR + '\\lib'
+elif os.name == 'osx':
+    if LIBKTX_INCLUDE_DIR is None:
+        LIBKTX_INCLUDE_DIR = '/usr/local/include'
+    if LIBKTX_LIB_DIR is None:
+        LIBKTX_LIB_DIR = '/usr/local/lib'
+
```

Comment:
What about defaults GNU/Linux?

Reply:
libktx installs itself the system header/library directory’s on GNU/Linux. That’s why explicitly specifying these isn’t necessary (and why the build passes on Linux CI).